packages/connectivity/src/scp-cf/xsuaa-service.ts

```diff
@@ -92,32 +63,19 @@ export async function getClientCredentialsToken(
         serviceCredentials,
         null,
         subdomainAndZoneId.zoneId,
-        (
-          err: Error,
-          token: string,
-          tokenResponse: ClientCredentialsResponse
-        ) => {
-          const isServiceInterface = (
-            serviceInterface: Service | string
-          ): serviceInterface is Service =>
-            !!(serviceInterface as Service).name;
-          const serviceName = isServiceInterface(service)
-            ? service.name
-            : service;
-
-          return err
-            ? reject(
-                `Error in fetching the token for service ${serviceName}: ${err.message}`
-              )
-            : resolve(tokenResponse);
-        }
+        (err: Error, token: string, tokenResponse: ClientCredentialsResponse) =>
+          err ? reject(err) : resolve(tokenResponse)
```

Comment:
[req] here we need to put back some part that only the message is included otherwise the whole axios config could end up in the logs again.